services/execution.go

```diff
@@ -261,7 +204,12 @@ func (es *executionService) constructBaseRunFromExecutable(executable state.Exec
 	// Compute the executable command based on the execution request. If the
 	// execution request did not specify an overriding command, use the computed
 	// `executableCmd` as the Run's Command.
-	executableCmd := executable.GetExecutableCommand(req)
+	executableCmd, err := executable.GetExecutableCommand(req)
```

Comment:
note: this function, `constructBaseRunFromExecutable` is invoked by both `es.CreateTemplateRun...` and `es.CreateDefinitionRun...`. Both structs implement the Executable interface, but `GetExecutableCommand` for a template is more complicated